docker/start.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3 -u
+
+# This replaces the old start.sh and ensures all arguments are bound correctly from the environment variables...
+import logging
+import os
+import subprocess
+import sys
+import time
+
+PYTHON3 = '/usr/bin/python3'
+argumentVariableMapping = {
+  '-l': 'LCID',
+  '-c': 'CLIENT_COUNT',
+  '-a': 'ACTIVATION_INTERVAL',
+  '-r': 'RENEWAL_INTERVAL',
+  '-w': 'HWID',
+  '-V': 'LOGLEVEL',
+  '-F': 'LOGFILE',
+  '-S': 'LOGSIZE',
+  '-e': 'EPID'
+}
+
+sqliteWebPath = '/home/sqlite_web/sqlite_web.py'
+enableSQLITE = os.path.isfile(sqliteWebPath) and os.environ.get('SQLITE', 'false').lower() == 'true'
+dbPath = os.path.join(os.sep, 'home', 'py-kms', 'db', 'pykms_database.db')
+log_level = os.getenv('LOGLEVEL', 'INFO')
+
+
+def start_kms_client():
+  if not os.path.isfile(dbPath):
+    # Start a dummy activation to ensure the database file is created
+    client_cmd = [PYTHON3, '-u', 'pykms_Client.py', os.environ.get('IP', "0.0.0.0"), os.environ.get('PORT', 1688),
```

Comment:
And also here https://github.com/Py-KMS-Organization/py-kms/commit/4ee742d671ae853f600b90425c97b2080ff17b80